README.md

```diff
@@ -27,15 +32,15 @@ Related tools added to image:
 
 OCM Container also includes multiple scripts for your ease of use. For a quick overview of what is available, run `list-utils`.
 
-## Quick Start:
+## Installation:
 
 * clone this repo
 * `./init.sh`
 * `./build.sh` (while this is building you can run the rest in another tab until the final command)
 * edit the file `$HOME/.config/ocm-container/env.source`
   * set your requested OCM_USER (for `ocm -u OCM_USER`)
   * set your OFFLINE_ACCESS_TOKEN (from [cloud.redhat.com](https://cloud.redhat.com/))
-* optional: add your PagerDuty API token in `~/.config/pagerduty-cli/config.json`
+* add your PagerDuty API token in `$HOME/.config/pagerduty-cli/config.json`
```

Comment:
Seeing that this isn't optional now, perhaps URL of how to generate a personal PD API key can be linked here as well for those of using ocm-container/PD for the first time. Might be useful for those who've been recently onboarded.

Reply:
it's still optional, I changed it as it's highly recommended.
I will return the optional